### DISCUSSION TITLE
Find email addresses for users in room

### DISCUSSION THREAD

**Discussion by oli414** [mozilla/hubs#4050] [Hubs Cloud Questions] [answered]
Hi there, is it possible to get a better insight in which username is associated with an e-mailaddress for the sake of tracking user activity? The plan is we are trying to get a better grip on our community and in example want to contact the right person for winning a contest.

**Comment by johnshaughnessy** [comment, marked as answer]
There's currently no way to do this as Hubs / Reticulum does not store email accounts. You will need to build your own service to do something like this and you may also need to provide users with a different privacy policy than the one we use on hubs.mozilla.com ( https://github.com/mozilla/hubs/blob/master/PRIVACY.md )
